### PR TITLE
cod1.c: remove old dead code

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

File: src/backend/cod1.c [hunk 1]
```diff
@@ -3539,18 +3522,9 @@ code *params(elem *e,unsigned stackalign)
...
-  else
-  {
-        WRTYxx(tym);
-        assert(0);
-  }
-  sz = align(stackalign,szb);           /* align on word stack boundary */
-  assert((sz & (stackalign - 1)) == 0); /* ensure that alignment worked */
+  targ_size_t szb = paramsize(e);               // size before alignment
+  targ_size_t sz = align(stackalign,szb);       // size after alignment
+  assert((sz & (stackalign - 1)) == 0);         // ensure that alignment worked
```
Comment (MartinNowak) [inline src/backend/cod1.c]: This is a rather paranoid assertion.

**Comment by MartinNowak** [comment]
> No description provided.

Please provide some more context, e.g. no longer need to handle OPparam b/c parameters are now flattened into an array beforehand.

**Comment by MartinNowak** [comment]
Auto-merge toggled on

**Comment by andralex** [comment]
noice

File: src/backend/cod1.c (hunk 1 above)
Comment (WalterBright) [inline src/backend/cod1.c]: I was tired of bugs with this :-)

**Pull request merged**